src/bundles/index.js

```diff
@@ -69,5 +70,10 @@ export default composeBundles(
   connectedBundle,
   retryInitBundle,
   ipfsDesktop,
-  repoStats
+  repoStats,
+  createAnalyticsBundle({
+    countlyAppKey: window.ipfsDesktop ? '47fbb3db3426d2ae32b3b65fe40c564063d8b55d' : '8fa213e6049bff23b08e5f5fbac89e7c27397612',
```

Comment:
nit: meaning can be lost/swapped during future refectoring, and it will be awfully hard to detect. 
May be safer to move API keys to `COUNTLY_APPKEY_DESKTOP` / `COUNTLY_APPKEY_WEBUI`  `const`s